Xamarin.Forms.Core/ToggleButton.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System;
+using System.ComponentModel;
+
```

Comment:
should be shared with Switch. it's fine to have multiple interface in a single file, but rename it to `ToggleButtonElement` then